docs/install.md

```diff
@@ -38,7 +47,15 @@ Then you can install MVT directly from [pypi](https://pypi.org/project/mvt/)
 pip install mvt
 ```
 
-Or from the source code:
+**Note:** In Arch, running the above command can result in python packages and linux packages being stored in same directory which can result in issues. To overcome this, use 
+
+```bash
+pip install --user mvt
```

Comment:
I would just recommend the `--user` everywhere, on mac too.

Reply:
Done